azure-pipelines.yml

```diff
@@ -29,7 +29,7 @@ stages:
             versionSpec: '16.14.0' # The version we're installing
         - task: Npm@1
           inputs:
-            command: 'install'
+            command: 'ci'
```

Comment:
We don't use package-lock file because it needs to be compatible with all major versions. Please check [the failure log](https://dev.azure.com/ng-zorro/NG-ZORRO/_build/results?buildId=5222&view=logs&j=279bf605-51e8-5634-b9b8-b38240bbf17e&t=f720c3be-c865-5389-3eca-d1d8af810617&l=4468)

Reply:
I remove this commit :) and pipeline is executed as before

Reply:
I suggest trying the datepicker or timepicker tests. those use the overlay, and it might be an issue if it's not cleaned up properly between tests.

Reply:
and/or in combination with async issues